CIScripts/Test/Tests/README.md

```diff
@@ -1,13 +1,20 @@
-# How to run tests from local machine
+# How to run tests from local machine on VMs in Juniper Windows CI Lab
 
 1. Deploy dev env using `deploy-dev-env` job.
 
-2. Install `powershel-yaml`:
+2. Install Pester:
+
+    ```Install-Module -Name Pester -Force -SkipPublisherCheck```
```

Comment:
I think it should be specifically Pester 4.2.0.
Pester 3.4.something is prebuilt into Windows AFAIK.